velox/connectors/hive/HiveDataSource.cpp

```diff
@@ -327,11 +328,8 @@ void HiveDataSource::setFromDataSource(
   VELOX_CHECK(source, "Bad DataSource type");
 
   split_ = std::move(source->split_);
-  if (source->splitReader_ && source->splitReader_->emptySplit()) {
-    runtimeStats_.skippedSplits += source->runtimeStats_.skippedSplits;
-    runtimeStats_.skippedSplitBytes += source->runtimeStats_.skippedSplitBytes;
-    return;
-  }
+  runtimeStats_.skippedSplits += source->runtimeStats_.skippedSplits;
```

Comment:
This is the fix?

Reply:
Yes, the fix is to not return on an empty split.

Reply:
This will set the `splitReader_` below.

Reply:
Thanks!